frontend/webpack.dev.js

```diff
@@ -21,5 +25,21 @@ module.exports = merge(common('development'), {
     client: {
       overlay: true,
     },
+    proxy: [
+      {
+        context: ['/api', '/api/llama-stack'],
```

Comment:
yeah the AI is right here - we only need `/api` to be proxied.
```suggestion
        context: ['/api'],
```